flake.nix

```diff
@@ -82,12 +83,16 @@
                     export HOME=~
                     export GOPATH=$PWD/go
                     export PATH=$GOPATH/bin:$PATH
+                    export PATH=''${CARGO_HOME:-~/.cargo}/bin:$PATH
```

Comment:
Since we've replaced `rust-overlay` with `rustup` as the management tool in Nix env, I would think the rust version is explicitly declared in `rust-toolchain.toml` no matter what outside of Nix is.

But yes, this line is unnessessary so just remove it.